File: Backend/Sistema de citas/Sistema de citas/Model/Servicios.cs
using System.ComponentModel.DataAnnotations;

namespace Sistema_de_citas.Model
{
    public class Servicios
    {
        public int Id { get; set; }
        public string service_names { get; set; }
        public string service_description { get; set; }
        public string duration { get; set; }
        public float price { get; set; }
    }
}

File: Frontend/Sistema_De_Citas/src/app/models/Servicios.ts
export class Servicios {
    id!: number;
    service_names!: string;
    service_description!: string;
    duration!: string;
    price!: number;

    constructor(
        id: number,
        service_names: string,
        service_description: string,
        duration: string,
        price: number
    ) {
        this.id = id;
        this.service_names = service_names;
        this.service_description = service_description;
        this.duration = duration;
        this.price = price;
    }
}